omnibus/config/templates/datadog-agent/upstart_debian.conf.erb

```diff
@@ -0,0 +1,23 @@
+description "Datadog Agent"
+
+start on started networking
+stop on runlevel [!2345]
+
+respawn
+
+# Logging to console from the agent is disabled since the agent already logs using file or
+# syslog depending on its configuration. We make upstart log what the process still outputs in order
+# to log panics/crashes to /var/log/upstart/datadog-agent.log
+console log
+env DD_LOG_TO_CONSOLE=false
+
+setuid dd-agent
+
+script
+  exec <%= install_dir %>/bin/agent/agent start -p <%= install_dir %>/run/agent.pid
+end script
+
+post-stop script
+  rm -f <%= install_dir %>/run/agent.pid
+  rm -f /tmp/agent.sock
```

Comment:
Unrelated to this PR, but we still create the UDS file?

Reply:
you're right, good catch, we don't create it anymore. I'll fix this in a different PR if you don't mind, this actually needs to be removed from a few other files too